### DISCUSSION TITLE
Hue Motion Sensor SML001 failing under zigbee2MQTT working on Hue Hub

### DISCUSSION THREAD

**Discussion by Busta999** [Koenkk/zigbee2mqtt#20644] [Devices]
**Device** 
Hue Motion Sensor Gen 1 - model [9290012607]
fw 6.1.1.27575
fw build 20190219

Over the last week 3 different Hue Gen 1 sensors have had this issue - so far tested

The issue is consistent and repeatable
The device is fully functioning under Hue Hub non functional under zigbee2MQTT 

So far three of the 17 hue Motion sensors under zigbee2MQTT have failed like this 2 in the last 12 hours.

Only Gen 1 Indoor Hue Motion Sensors impacted so far

**zigbee2MQTT** 
version 1.35.1
Coordinator zStack3x0 version 20230507

**Symptoms**
Sensors are reported as Online

Hue Motion Sensor stops reporting :
- [ ] Occupancy
- [ ] Temperature
- [ ] Battery
- [ ] illuminance
- [ ] lux

With Led Indication set to ON - the Motion sensor Led flashes on detecting motion - but nothing is reported in zigbee2MQTT

Replacing batteries - no change
Re- Pairing - no change

Remove from zigbee2MQTT  and add to Hue Hub - devices starts functioning again and is fully functioning in HASS through Hue integration

Reboot HASS VM

Remove from Hue and re-add to zigbee2MQTT - same result devices connects - restoring original Friendly Name but with no Temp/Lux/battery/occupancy updates

**Comment by Kuchiru** [comment]
I have the same issue with a Gen1 motion sensor.

Firmware version 6.1.1.27575
Firmware build date 20190219
Z2M: 1.42.0
Hardware: EmberZNet 7.4.5 [GA]

The motion sensor is consistently stuck on all values it first reported and will not report any new changes just like @Busta999
Re-Pairing: no change
Batteries replaced: no change

Works with the ZHA integration without issue.

**Comment by MarvAmBass** [comment]
Same Problem here, but for me it suddenly stopped working.

Battery Change / Reset are no solution

It worked for a few days after a fresh firmware update and stopped working

Firmware ID
67.115.5
20240524

**Comment by MarvAmBass** [comment]
Update:

I've set

Motion sensitivity Medium -> High
Occupancy timeout 0 -> 2s
Led indication True -> False

And clicked the Reconfigure Button in Zigbee2MQTT, I also did a Zigbee2MQTT Upgrade
```
Zigbee2MQTT version
[2.8.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/2.8.0)
commit: [b0b02c30b2e58c94043298c14848c90a0f74a24a](https://github.com/Koenkk/zigbee2mqtt/commit/b0b02c30b2e58c94043298c14848c90a0f74a24a)
Frontend version
[2.8.1](https://github.com/Nerivec/zigbee2mqtt-windfront/releases/tag/v2.8.1)
zigbee-herdsman-converters version
[25.120.0](https://github.com/Koenkk/zigbee-herdsman-converters/releases/tag/v25.120.0)
zigbee-herdsman version
[9.0.2](https://github.com/Koenkk/zigbee-herdsman/releases/tag/v9.0.2)
```

and now it seems to work - I guess that the `Occupancy timeout 0` might have been the problem - I remember I set this by accident somehow
unfortuantely I didn't have the time to test one after another

if it fails again in a few days I'll update the issue - if not it should have worked
